bigtable/client/internal/instance_admin_test.cc

```diff
@@ -259,6 +259,7 @@ TEST_F(InstanceAdminTest, ListInstancesUnrecoverableFailures) {
   grpc::Status status;
   tested.ListInstances(status);
   EXPECT_FALSE(status.ok());
+  EXPECT_NE(std::string::npos, status.error_message().find("uh oh"));
```

Comment:
If you prefer GMock, how about:

```cpp
EXPECT_THAT(status.error_message(), Contains("uh oh"));
```

Simpler and more readable, but adds a dependency and a top-level `using` declaration. Thoughts?